website/docs/d/spatial_anchors_account.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Mixed Reality"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spatial_anchors_account"
+description: |-
+  Get information about an Azure Spatial Anchors Account.
+---
+
+# azurerm_spatial_anchors_account
+
+Get information about an Azure Spatial Anchors Account.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
```

Comment:
Can we modify the example a bit to only includes the `data` block and the `output` block ([example](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/virtual_network))?